### PR TITLE
Improve run script and fix small binding bug on precompile template

### PR DESCRIPTION
**Pull request by anusha-ctrl**
-  Fix small binding bug on the precompile template

### CODE REVIEW

File: accounts/abi/bind/bind.go [hunk 1]
```diff
@@ -658,7 +658,7 @@ func decapitalise(input string) string {
 func convertToNil(input abi.Type) string {
 	switch input.T {
 	case abi.IntTy, abi.UintTy:
-		return "0"
+		return "big.NewInt(0)"
```
Comment (aaronbuchwald) [inline accounts/abi/bind/bind.go:661]: Nice catch!

File: scripts/run.sh [hunk 2]
```diff
@@ -40,7 +40,7 @@ ENABLE_SOLIDITY_TESTS=${ENABLE_SOLIDITY_TESTS:-false}
 AVALANCHE_LOG_LEVEL=${AVALANCHE_LOG_LEVEL:-INFO}
 ANR_VERSION=$network_runner_version
 GINKGO_VERSION=$ginkgo_version
-
+GINKGO_FOCUS_FLAGS=${GINKGO_FOCUS_FLAGS}
```
Comment (aaronbuchwald) [inline scripts/run.sh:43]: Can we make this a separate PR?

**Comment by aaronbuchwald** [comment]
Can we separate this into two PRs?

File: scripts/run.sh (hunk 2 above)
Comment (anusha-ctrl) [reply scripts/run.sh]: done

File: accounts/abi/bind/bind.go (hunk 1 above)
Comment (anusha-ctrl) [reply accounts/abi/bind/bind.go:661]: thanks!

**Review by aaronbuchwald — APPROVED** [review]
LGTM
